apparmor.d/abstractions/nvidia.d/complete

```diff
@@ -5,3 +5,7 @@
   owner @{user_cache_dirs}/nvidia/ w,
   owner @{user_cache_dirs}/nvidia/GLCache/ rw,
   owner @{user_cache_dirs}/nvidia/GLCache/** rwk,
+
+  @{run}/nvidia-xdriver-* w,
```

Comment:
Should this been put in this abstraction? Across all the profiles it is only found in the `xorg` one.

Reply:
It may be tied to hardware acceleration. I will recheck if it works without this rule.

Reply:
Can't see any difference. Removed.